### ISSUE TITLE
Vue DevTools permissions issue

### ISSUE THREAD

**Issue by nwittwer** [michalzaq12/electron-nuxt#654]
Error:
```
[33492:0402/121823.078:ERROR:CONSOLE(1)] "Extension server error: Operation failed: Permission denied", source: devtools://devtools/bundled/extensions/ex failed: Permistensions.js (1)
```

[Line 20](https://github.com/michalzaq12/electron-nuxt/blob/master/template/src/main/boot/index.dev.js#L20) seems to be triggering this issue about "Permission denied" when installing the Vue devtools electron plugin:
```js
installExtension(VUEJS_DEVTOOLS)
```

The error message disappears if you disable [line 71](https://github.com/michalzaq12/electron-nuxt/blob/master/template/src/main/BrowserWinHandler.js#L71), which is responsible for opening the DevTools
```js
onCreated(callback) {
    if (this.browserWindow !== null) return callback(this.browserWindow)
    this._eventEmitter.once('created', () => {
      callback(this.browserWindow)
      // if (isDev && BrowserWindow.getAllWindows().length === 1) this.browserWindow.webContents.openDevTools()    <-- Disable this line
    })
  }
```

This makes me think that maybe the devtools are being installed at the incorrect time, before the DevTools is available? Has anyone else run into this issue?

**Comment by donxp** [comment]
Same issue here, if I comment out line 71 I get no errors in the console, and when I open devtools the vue extension is present.

**Comment by michalzaq12** [comment]
Issue should be fixed in 1.7.1 https://github.com/michalzaq12/electron-nuxt/commit/cbf95cc0cf0d2ed0fe4b0c9d60c2f5b4ef93a750
